engine/src/main.cpp

```diff
@@ -671,16 +663,16 @@ void bootstrap_main_loop() {
         UpdateTime();
         FactionUtil::LoadContrabandLists();
         {
-            if (!game_options.intro1.empty()) {
-                UniverseUtil::IOmessage(0, "game", "all", game_options.intro1);
-                if (!game_options.intro2.empty()) {
-                    UniverseUtil::IOmessage(4, "game", "all", game_options.intro2);
-                    if (!game_options.intro3.empty()) {
-                        UniverseUtil::IOmessage(8, "game", "all", game_options.intro3);
-                        if (!game_options.intro4.empty()) {
-                            UniverseUtil::IOmessage(12, "game", "all", game_options.intro4);
-                            if (!game_options.intro5.empty()) {
-                                UniverseUtil::IOmessage(16, "game", "all", game_options.intro5);
+            if (!game_options()->intro1.empty()) {
```

Comment:
This calls for some elegant refactoring. Something like:
for {
    bla "intro"+i
}
If you don't want to, just put a TODO and we'll get to it in due course.

Reply:
OK, I'll look into that.